testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/Environment.java

```diff
@@ -685,7 +706,7 @@ private static Consumer<OutputFrame> combineConsumers(Consumer<OutputFrame>... c
 
         private void setContainerOutputConsumer(Function<DockerContainer, Consumer<OutputFrame>> consumer)
         {
-            configureContainers(container -> container.withLogConsumer(consumer.apply(container)));
+            configureContainers(dockerContainer -> true, container -> container.withLogConsumer(consumer.apply(container)));
```

Comment:
nit: There are only two occurrences where the predicate is `true` but maybe add an override for this to make this a little bit easier to read.